filesystem/etc/calico/confd/templates/bird.cfg.template

```diff
@@ -39,6 +40,7 @@ protocol direct {
   interface -"cali*", "*"; # Exclude cali* but include everything else.
 }
 
+{{if ne "" ($node_ip)}}# IPv4 disabled on this node.
```

Comment:
Is this comment in the right place? Won't that show up even when node_ip is defined?